tf-static-website/22-cloudfront-frontend.tf

```diff
@@ -30,7 +30,7 @@ resource "aws_cloudfront_distribution" "s3_distribution" {
   aliases = [ var.domain_name ]
 
   default_cache_behavior {
-    allowed_methods  = ["GET", "HEAD", "POST", "OPTIONS"]
+    allowed_methods  = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
```

Comment:
FYI: During terraforming, I got the error below:
```
The parameter AllowedMethods is invalid (must be: [HEAD, GET] or [HEAD, GET, OPTIONS] or [HEAD, DELETE, POST, GET, OPTIONS, PUT, PATCH]).
```

So, I choose the full method option.